src/main/java/Entities/Timeblockable.java

```diff
@@ -0,0 +1,29 @@
+package Entities;
+
+public interface Timeblockable {
+    int[] timeBlock = new int[2];
```

Comment:
Again - unless this is a static variable or a constant, this instance attribute needs to be moved out of this interface

Reply:
Fixed issues in both interfaces